integration-tests/e2e-tests/src/abilities/WalletSdk.ts

```diff
@@ -84,33 +94,10 @@ export class WalletSdk extends Ability implements Initialisable, Discardable {
   }
 
   isInitialised(): boolean {
-    return this.sdk.state != "stopped"
-  }
-}
-
-class WalletSdkBuilder {
```

Comment:
Here I've simplified the ability build, I noticed it was too complex. The `initialise` method is called when the actor uses the ability, so now on it will create the sdk object when it's called (not before!)